pkg/logs/writer.go

```diff
@@ -29,10 +35,27 @@ type BufferedLog struct {
 // NewBufferedWriter returns an io.Writer that writes log chunk messages to the gRPC sender for the
 // named Tekton result. The chunk size determines the maximum size of a single sent message - if
 // less than zero, this defaults to DefaultBufferSize.
-func NewBufferedWriter(sender Sender, name string, size int) *BufferedLog {
+func NewBufferedWriter(sender LogSender, name string, size int) *BufferedLog {
```

Comment:
Where do we use this function? Aren't we sending using NewBufferedHTTPWriter only?

Reply:
Yes, are using in the watcher because the `UpdateLog` client still uses the `Log` message def. 

https://github.com/tektoncd/results/blob/0aa7ce4900d905dfac265bd8542e99bda451d4c7/pkg/watcher/reconciler/dynamic/dynamic.go#L353

https://github.com/tektoncd/results/blob/0aa7ce4900d905dfac265bd8542e99bda451d4c7/proto/v1alpha2/api.proto#L111